### ISSUE TITLE
Firefly wallet v1.0.0 installed but opens as blank page¡

### ISSUE THREAD

**Issue by TermiteJM** [iotaledger/firefly#1070]
I downloaded the Firefly wallet v 1.0.0 on 28th April to my Linux Mint but it opens to a completely blank page.

**Comment by m4c4d1** [comment]
Same for me on macOSX 10.13.6

**Comment by Hagen2012** [comment]
Same for me, using MacOS High Sierra version 10.13.6

**Comment by eKing021** [comment]
same
Mac 10.13.6
Firefly 1.0.0

**Comment by rajivshah3** [comment]
Hi, can you please try the latest version and post any error messages that are shown?

**Comment by m4c4d1** [comment]
works fine for me now with version 1.1.0 on macOSX 10.13.6. Just waiting for Ledger support now.

**Comment by Hagen2012** [comment]
Rajiv

Thanks, seems to work, but I installed firefly on my mac notebook without any problem (MacOs Catalina 10.15.6).  Can I also install or copy the wallet to my desktop?

Kind regards
Hagen

On 1 May 2021, at 00:24, Rajiv Shah ***@***.***> wrote:


Hi, can you please try the latest version and post any error messages that are shown?

**Comment by rajivshah3** [comment]
Yes, you can use it on multiple devices. You just need to install Firefly and import your wallet backup on the other computer
